universal-application-tool-0.0.1/test/services/applicant/ApplicantDataTest.java

```diff
@@ -176,6 +226,19 @@ public void readString_findsCorrectValue() throws Exception {
     assertThat(found).hasValue("orange");
   }
 
+  @Test
+  public void readString_withRepeatedEntity_findsCorrectValue() throws Exception {
+    ApplicantData data =
+        new ApplicantData(
+            "{\"applicant\":{\"children\":[{\"entity\":\"first child\", \"name\": { \"first\":"
```

Comment:
same super nit - maybe put the  entity 0 info on the same line?

Reply:
done

Reply:
done!